gemini-2/live_api_starter.py

```diff
@@ -27,42 +26,36 @@
 from google import genai
 
 if sys.version_info < (3, 11, 0):
-    import taskgroup, exceptiongroup
-    asyncio.TaskGroup = taskgroup.TaskGroup
-    asyncio.ExceptionGroup = exceptiongroup.ExceptionGroup
+    raise EnvironmentError("This script requires python 3.11+")
```

Comment:
Maybe revert this depending on if https://github.com/google-gemini/cookbook/pull/353 solves it.

Reply:
Done.